packages/e2e-tests/.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
```

Comment:
Does this file do anything? I thought `.github` is supposed to be at the top level of the repo.

Reply:
It's an auto generated github action that we will integrate in another step later